compiler/dcalc/from_scopelang.ml

```diff
@@ -914,6 +916,17 @@ let translate_scope_decl
         | _ -> ctx)
       ctx scope_variables
   in
+  let date_rounding =
+    List.find_opt
+      (function Desugared.Ast.DateRounding _ -> true)
+      sigma.scope_options
+    |> Option.map (function
+         | Desugared.Ast.DateRounding Desugared.Ast.Increasing ->
+           (RoundUp : Runtime_ocaml.Runtime.date_rounding)
```

Comment:
```suggestion
           Runtime_ocaml.Runtime.RoundUp
```
is the more classic syntax, more concise and doesn't rely on type-based disambiguation